packages/contracts-bedrock/contracts/universal/ProxyAdmin.sol

```diff
@@ -147,13 +147,13 @@ contract ProxyAdmin is Owned {
      * @return The address of the implementation.
      */
     function getProxyImplementation(Proxy proxy) external view returns (address) {
-        ProxyType proxyType = proxyType[address(proxy)];
+        ProxyType _proxyType = proxyType[address(proxy)];
 
-        if (proxyType == ProxyType.ERC1967) {
+        if (_proxyType == ProxyType.ERC1967) {
```

Comment:
this is a compiler warning?

Reply:
Its shadowing another variable so renaming it with the `_` prefix fixes that